src/firefly/js/visualize/saga/MouseReadoutWatch.js

```diff
@@ -235,15 +241,22 @@ function showSingleBandFluxLabel(plot, band) {
 
 
 function doFluxCall(plotView,iPt) {
-    const plotStateAry= getPlotStateAry(plotView);
-    return callGetFileFlux(plotStateAry, iPt)
-        .then((result) => {
-            return result;
-        })
-        .catch((e) => {
-            console.log(`flux error: ${plotView.plotId}`, e);
-            return ['', '', ''];
-        });
+    const plot= primePlot(plotView);
+    const {x,y}= iPt;
+    if (x >=0 && y >= 0 && x < plot.dataWidth && y < plot.dataHeight) {
```

Comment:
is is support to do the same as line 140.  I meant to fix both. It check to see it the image point is on the plot. It should call `pointInPlot`